src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TermQueryExplainIT.java

```diff
@@ -102,6 +102,7 @@ public void testNonResolvingIndexPatternWithNonExistingIndex() throws IOExceptio
     }
 
     @Test
+    @Ignore("allow non-identical mapping when query multi-index")
```

Comment:
I prefer deleting the tests that are not asserting any truth anymore.

Reply:
Deleted